### PR TITLE
add new html and add copy

### PR DESCRIPTION
**Pull request by e-moran**
# Why
Adds more customisation to Reveal.
Adds a Copy button to Reveal
Fixed a bug where Inputs wouldn't encrypt in Debug mode for free teams

# How
Use a new HTML page for Reveal so that its completely separate from Inputs

### CODE REVIEW

File: packages/browser/README.md [hunk 1]
```diff
@@ -185,6 +185,57 @@ inputs.isInputsLoaded.then(() => {
...
+Once you have your endpoint that returns the encrypted card data, you'll need to create an
+[Evervault Inbound Relay](https://docs.evervault.com/products/inbound-relay) that will decrypt the
+encrypted card data as is passes through it, before it gets to the iFrame. Once you have created
+your Relay you can add the component to your React app.
+
+```html
+<div id="reveal"></div>
+
+<script>
+  let request = new Request("https://secure.example.com", {
```
Comment (thomascullen) [inline packages/browser/README.md:215]: Should we change this to something like `https://my-endpoint.relay.evervault.com` to indicate it should be the relay endpoint?

File: packages/browser/lib/core/inputs.ts [hunk 2]
```diff
@@ -38,6 +42,7 @@ export default function Inputs(config: Config) {
 
       const isInputsLoaded = new Promise<boolean>((resolve) => {
         window.addEventListener("message", (event) => {
+          console.log(event);
```
Comment (thomascullen) [inline packages/browser/lib/core/inputs.ts:45]: Remove `console.log`?

File: packages/browser/lib/utils/constructSource.ts [hunk 3]
```diff
@@ -21,9 +24,14 @@ export default function constructSource(
...
+  let { inputsOrigin } = config.input;
   const { teamId, appId } = config;
 
+  let inputsUrl = inputsOrigin + INPUTS_URL_SLUG;
+  if (isReveal) {
+    inputsUrl = inputsOrigin + REVEAL_URL_SLUG;
+  }
+
   return `${inputsUrl}?team=${teamId}&app=${appId}${settingsQuery}&mode=${
     isReveal ? "reveal" : "inputs"
```
Comment (thomascullen) [inline packages/browser/lib/utils/constructSource.ts:36]: Do we still need to pass mode if we have a separate html file for reveal now?

**Review by thomascullen — CHANGES_REQUESTED** [review]
Nice! 👏 Couple of small nits. Ping me when ready for review again and will do full QA.

File: packages/browser/README.md (hunk 1 above)
Comment (e-moran) [reply packages/browser/README.md:215]: Great idea!

File: packages/browser/lib/core/inputs.ts (hunk 2 above)
Comment (e-moran) [reply packages/browser/lib/core/inputs.ts:45]: Yes, nice catch!

**Review by thomascullen — APPROVED** [review]
Played around locally and all seems to work! Nice job 👏

**Pull request merged**